examples/demo/src/models/users.rs

```diff
@@ -216,12 +216,10 @@ impl super::_entities::users::Model {
     /// # Errors
     ///
     /// when could not convert user claims to jwt token
-    pub fn generate_jwt(&self, secret: &str, expiration: &u64) -> ModelResult<String> {
-        Ok(jwt::JWT::new(secret).generate_token(
-            expiration,
-            self.pid.to_string(),
-            Some(json!({"Roll": "Administrator"})),
-        )?)
+    pub fn generate_jwt(&self, secret: &str, expiration: u64) -> ModelResult<String> {
+        let mut claims = Map::new();
+        claims.insert("Roll".to_string(), "Administrator".into());
```

Comment:
it seems that there was a typo in this line before

should I change it to
```suggestion
        claims.insert("Role".to_string(), "Administrator".into());
```?

Reply:
Think so yeah, can't imagine 'roll' was the intended term here haha